tools/tsp-client/src/log.ts

```diff
@@ -55,26 +55,32 @@ export function printBanner() {
 }
 
 const usageText = `
-Usage: tsp-client [options]
+Usage: tsp-client <command> [options]
 
-Generate from a tsp file using --mainFile or use tsp-location.yaml inside 
-the outputDir.
+Use one of the supported commands to get started generating clients from a TypeSpec project.
+This tool will default to using your current working directory to generate clients in and will
+use it to look for relevant configuration files. To specify a different directory, use
+the -o or --output-dir option.
 
-Positionals:
+Commands:
   init        Initialize the SDK project folder from a tspconfig.yaml   [string]
-  sync        Sync tsp files using tsp-location.yaml                    [string]
-  generate    Generate from a tsp project                               [string]
-  update      Sync and generate from a tsp project                      [string]
+  sync        Sync TypeSpec project specified in tsp-location.yaml      [string]
+  generate    Generate from a TypeSpec project                          [string]
+  update      Sync and generate from a TypeSpec project                 [string]
 
 Options:
-  -d, --debug      Enable debug logging                                [boolean]
-  -e, --emitter    Which language emitter to use             [required] [string]
-                  [choices: "csharp", "java", "javascript", "python", "openapi"]
-  -m, --mainFile   The url of the main tsp file to generate from        [string]
```

Comment:
a little sad that we're getting rid of generating from a remote tsp, but I guess I was the only one who thought that feature was cool?

Reply:
Did you do this as opposed to the tspconfig? I think we still support remote tspconfig files correct?

Reply:
We do support both local paths and remote urls, for the `init` command you can specify the `--tsp-config` option and we'll check if it's a remote url or local path and act accordingly. This tool currently aims to match the behavior of the powershell scripts so we should be able to do everything that the powershell scripts can do.

Reply:
Later on, I want to add some sort of `main-file` or `entry-file` option to account for entry files that arent following standard names like main.tsp or client.tsp